lib/straight/blockchain_adapters_dispatcher.rb

```diff
@@ -34,8 +34,12 @@ def execute_in_parallel(block, adapters)
       attempts_counter = Concurrent::MVar.new(adapters_to_run.size)
       pool             = Concurrent::ThreadPoolExecutor.new
       adapters_to_run.each do |adapter|
-        p = Concurrent::Promise.new(executor: pool) { block.call(adapter) }
+        p = Concurrent::Promise.new(executor: pool) {
```

Comment:
It depends. The second convention makes more sense to me:

>Use curly brackets for functional blocks, where the primary purpose of the block is to return a value.
Use do...end for procedural blocks, where the primary purpose of the block is its side-effects. 

http://devblog.avdi.org/2011/07/26/the-procedurefunction-block-convention-in-ruby/